src/redux/missions/missions.js

```diff
@@ -2,8 +2,24 @@ const ADD_MISSION = 'spaceHub/missions/ADD_MISSION';
 const REMOVE_MISSION = 'spaceHub/missions/REMOVE_MISSION';
 const GET_MISSIONS = 'spaceHub/missions/GET_MISSIONS';
 
+const API_URL = 'https://api.spacexdata.com/v3/missions';
+
 const initialState = [];
 
+export const getMissions = () => async (dispatch) => {
+  let missions = [];
+  await fetch(API_URL)
+    .then((response) => response.json())
+    .then((data) => {
+      missions = data;
+    });
+  console.log('missions', missions);
```

Comment:
- [ ] OPTIONAL
please be sure to delete the console

Reply:
I forgot to delete this, fixed.